vcd_cli/role.py

```diff
@@ -51,19 +54,48 @@ def list_roles(ctx):
               '--org',
               'org_name',
               required=False,
-              metavar='<org-name>',
+              metavar='<org_name>',
```

Comment:
metavar shouldn't have _

Reply:
done. Got confused from our conversation other day, though it should not have - not _